src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -486,6 +492,7 @@ public GHRepository getRepository(String name) throws IOException {
     @Preview @Deprecated
     public PagedIterable<GHLicense> listLicenses() throws IOException {
         return new PagedIterable<GHLicense>() {
+            @Override
             public PagedIterator<GHLicense> _iterator(int pageSize) {
```

Comment:
Added `@Override` annotation to overridden methods as a matter of lint/best-practice.